department-of-veterans-affairs/va.gov-cms
# VAGOV-5807: Fix Owner dropdown on bulk edit view.

QA:
* Go to /admin/content/bulk
* Filter by Owner field a few times. it should work as expected... the Owner field in the filtered results should reflect whatever you choose from the dropdown. When you filter by parent terms (eg Veterans Benefits Administration) it should show content tagged with VBA's child terms too.
